functions/index.js

```diff
@@ -1 +1,48 @@
+const fs = require('fs');
+const fetch = require('node-fetch');
+const path = require('path');
+const os = require('os');
+const env = require('dotenv').config();
```

Comment:
Check out https://firebase.google.com/docs/functions/config-env#retrieve_current_environment_configuration
I think we can avoid using `.env`

Also check out my following comment, maybe we can avoid any of this